packages/application-components/src/components/project-stamp/project-stamp.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from '@emotion/react';
+import { useIntl } from 'react-intl';
+import { DotIcon } from '@commercetools-uikit/icons';
+import Stamp, { TTone } from '@commercetools-uikit/stamp';
+import messages from './messages';
+
+type TProjectStampProps = {
+  isProductionProject?: boolean | undefined;
+  isSuspended?: boolean | undefined;
+  isExpired?: boolean | undefined;
+  willExpire?: boolean | undefined;
+  daysLeft?: number | undefined;
```

Comment:
Adding the question mark beside the property name already make the its type to be potentially `undefined` so we don't need to add it in the definition type.